### ISSUE TITLE
bug with BpR count on char creation

### ISSUE THREAD

**Issue by igroglaz** [draconisPW/PWMAngband#587]
https://youtu.be/_9JLJXQg7ug

**Comment by igroglaz** [comment]
it happened to 

https://discord.com/channels/@me/516688804799578113/1066701520252436561

villager beholder.. so it's pretty strange bug. Villager has min-weight:35

**Comment by draconisPW** [comment]
cidx = 38 corresponds to the "Ghost" class in Tangaria's class.txt file. Sounds like a class was deleted without a wipe and char got remapped to ghost instead.

**Comment by igroglaz** [comment]
yeah, thanks :) classes counts from 0, so it's ghost indeed! **BUT...** we didn't have any new classes or class deletions. I didn't touch classes numbers for a long time. So it's different bug :(
